### PR TITLE
Update site mobile header

### PR DESCRIPTION
**Pull request by angel**
Updated the styling of the mobile header nav to look more like what had been mocked up for the header and nav component. This is what it looks like:

image

### CODE REVIEW

**Comment by maya** [comment]
@angel you need to make sure it works for desktop. Now it looks like this:

screen shot 2015-09-18 at 10 21 27 am

File: assets/css/styleguide.scss [hunk 1]
```diff
@@ -128,7 +130,16 @@ $site-top:           124px;
   display: inline;
   float: left;
   margin-top: -4px;
-  padding-left: 20px;
+  padding-top: 15px;
```
Comment (maya) [inline assets/css/styleguide.scss]: Lets use rems for anything more than a few pixels per the 18F coding styleguide.

Comment (angel) [inline assets/css/styleguide.scss]: @maya Fixed! Sorry should not do a PR so late! :P Let me know if I missed anything else!

**Comment by angel** [comment]
@maya Fixed! Sorry should not do a PR so late! :P Let me know if I missed anything else!

File: assets/css/styleguide.scss [hunk 2]
```diff
@@ -128,7 +132,23 @@ $site-top:           124px;
   display: inline;
   float: left;
   margin-top: -4px;
-  padding-left: 20px;
+  padding-top: 1.5rem;
+  padding-bottom:1.5rem;
```
Comment (maya) [inline assets/css/styleguide.scss]: please put a space after the `:`

Comment (angel) [inline assets/css/styleguide.scss]: Added!

**Comment by maya** [comment]
lgtm :+1:

**Pull request merged**